src/TicketService.java

```diff
@@ -29,11 +31,30 @@ public static void main(String[] args) {
         for (Ticket ticket : ticketsBySector) {
             System.out.println(ticket);
         }
+
+        Client client = new Client(1);
```

Comment:
please, use abstraction, not realisation 
User client = new Client()

Reply:
If I use abstraction , I can't use method getTicket() from  Client class.  Do I need to create new interface with this method ?

Reply:
((Client) client).getTicket() ; works?

Reply:
Sorry! I forgot about casting (( Thank you